hardware/raspberrypi/bootmodes/nvme.md

```diff
@@ -71,13 +70,13 @@ sudo mount /dev/nvme0n1p2 mnt/ext4
 sudo ROOT_PATH=mnt/ext4 BOOT_PATH=mnt/fat32 rpi-update
 ```
 
-Finally, if you are using CM4lite remove the SD card and see if the board boots from the NVMe disk. For versions of CM4 with an eMMC you have to make sure the NVMe is first in the boot order.
+Finally, if you are using CM4 lite, remove the SD card and the board will boot from the NVMe disk. For versions of CM4 with an eMMC, make sure you have set NVMe first in the boot order.
```

Comment:
Slight duplication here with an earlier sentence?

Reply:
> Slight duplication here with an earlier sentence?

There is, but it probably doesn't hurt to mention this potential gotcha for a second time. I would suggest that's probably a question for Tim - apart from anything else, he wrote the page with 2 mentions in it.

Reply:
Just to be pedantic, this wasn't written by Tim :wink:  #1840

Reply:
> Just to be pedantic, this page wasn't written by Tim 😉 #1840

I see. My point was mainly that Tim is better placed than I am to decide on what level of detail, repitition etc is best when it comes to documentation about the bootloader's features.